---
bool success
string message
dcsc_setups/MopsSensors sensors
